src/applications/disability-benefits/4142/helpers.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+export const aboutPrivateMedicalRecs = () => {
+  return (
+    <div>
+      <h4>About private medical records</h4>
+      <p>
+        You said you were treated for [condition] by a private doctor. If you
+        have those records, you can upload them here, or we can get them for
+        you. If you want us to get your records, you'll need to authorize their
```

Comment:
Apostrophes are typically curly `‘`.

Reply:
Apostrophes are hard. I _think_ that one should be `’` (unicode 2019), not `‘` (unicode 2018). :man_shrugging:
That's what I've been using, anyhow.